### DISCUSSION TITLE
External files (CSS, Images & JavaScript) not working in my phone.

### DISCUSSION THREAD

**Discussion by pawanweber** [community/community#69791] [Programming Help]
### Body

Hi Sir,
I'm facing an issue in my code. I have designed a website using HTML, CSS, and JavaScript, and I've made it responsive with the help of media queries.
When I run the index file on my laptop or on someone else's laptop, it looks good, and all the functions are working well.
However, when I open the same file on a mobile device, only the HTML file is loading. It seems that not all the external files (CSS, image folder, JavaScript) are supported on the phone.
But when I paste the CSS and JavaScript code directly into the HTML file (index.html) and run it on my phone, CSS and JavaScript support seems to work.
I have linked the CSS and JavaScript files properly with the HTML file using the correct syntax and also added important meta tags.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebubekirgungor** [comment]
Hello, can you share the link of your website?

**Reply by pawanweber** [reply]
It's not a live project, I'm working in my local pc.

**Reply by ebubekirgungor** [reply]
Go to `view-source:http://your-website/your-index.html` on your mobile device and check the external file links.

**Reply by pawanweber** [reply]
https://pawanweber.github.io/Astro.github.io/
When I make it live on GitHub, it works well on all mobile devices. However, when I try to run this file as a local file on my phone, external files are not supported at that time.

**Reply by ebubekirgungor** [reply]
Maybe it is about cookies. Can you try in incognito tab?

**Comment by fredschuck** [comment]
I want to say that the issue you're experiencing could be related to how you're accessing the files on your phone when opening them locally. Seems like a path issue. How are you opening it exactly?
